libraries/AC_Fence/AC_Fence.h

```diff
@@ -57,6 +58,9 @@ class AC_Fence
 
     void init() {
         _poly_loader.init();
+        AP_Param::set_and_save_by_name("FENCE_ALT_MAX", AC_FENCE_ALT_MAX_DEFAULT);
+        AP_Param::set_and_save_by_name("FENCE_ALT_FRAME", AC_FENCE_ALT_FRAME_DEFAULT);
+        AP_Param::set_and_save_by_name("FENCE_ALT_MIN", AC_FENCE_ALT_MIN_DEFAULT);
```

Comment:
I'm not sure we should be re-setting these. Seams reasonable to persist, 'normal' fences do.